geode-core/src/main/java/org/apache/geode/internal/cache/wan/GatewaySenderEventImpl.java

```diff
@@ -1354,4 +1355,20 @@ protected GatewaySenderEventImpl makeCopy() {
   public void setAcked(boolean acked) {
     isAcked = acked;
   }
+
+  public enum TransactionMetadataDisposition {
+    /**
+     * Transaction metadata should be excluded from the event.
+     */
+    Exclude,
```

Comment:
I think the geode style guide says constants should be all upper case. The members of an enum are constants. See https://google.github.io/styleguide/javaguide.html#s5.2-specific-identifier-names
I think you should follow this convention for this new enum

Reply:
Yes, you are correct. :(